pepys_admin/maintenance/column_data.py

```diff
@@ -220,11 +221,8 @@ def create_normal_column_data(col, data_store, table_object):
     # Skip getting values for the remarks column, as we don't need a dropdown for that
     if details["type"] == "string" and details["system_name"] != "remarks":
         # Get values
-
-        all_records = data_store.session.query(table_object).all()
-        values = [
-            str_if_not_none(getattr(record, details["system_name"])) for record in all_records
-        ]
+        all_records = data_store.session.query(getattr(table_object, details["system_name"])).all()
```

Comment:
Just to help out our future-selves could you explain in more detail what is happening above?  I understand the PR description, but don't quite see how the above line addresses the change.  It may be worth inserting a comment on the previous line.

Reply:
Done

Reply:
Aah - I now understand where the performance increase came from :-)